libs/openchallenges/home/src/lib/home.component.html

```diff
@@ -3,13 +3,14 @@
     <h4>Welcome to OpenChallenges!</h4>
     <h3>This website uses a preview snapshot of the OpenChallenges (OC) Database</h3>
     <h4>
-      We are working hard to update the OC Database with more data in the upcoming months. A more
-      comprehensive database will be released in September 2023. We encourage organizations to
+      We are working hard to update the OC Database with more data in the upcoming months, and we
+      need your help! We encourage organizations and participants to
       <a
         href="https://docs.google.com/forms/d/e/1FAIpQLSdLZ6C5ZIzLZS147bmvNm2eJ3kq_88ecHS7ADjD4KaapGhO8g/viewform"
         target="_blank"
         >submit new challenges</a>
-      and update existing ones to ensure their profiles are complete.
+        and update existing ones. This will help ensure that organization profiles are complete and
+        that OC users can access accurate and up-to-date challenge resources.
```

Comment:
What about:

> Contribute to the OC Database and help us make it even better! We invite organizations and participants to submit new challenge and update existing ones.  This will ensure that organization profiles are complete and OC users have access to accurate and up-to-date resources.

Or something similar to sound more inviting/encouraging about data contributions?

Reply:
I opened a new PR since I like the suggestion. See https://github.com/Sage-Bionetworks/sage-monorepo/pull/2190